src/ints/mouse.cpp

```diff
@@ -1099,6 +1190,135 @@ Bitu UIR_Handler(void) {
+
+    MouseVMW_NewScreenParams(x_abs, y_abs);
+}
+
+void Mouse_EventMoved(int32_t x_rel, int32_t y_rel, int32_t x_abs, int32_t y_abs, bool is_captured) {
+
+	MouseVMW_NotifyMoved(x_abs, y_abs);
+
+    // TODO: change the code so that in seamless mode sensitivity is not applied
+	Mouse_CursorMoved(x_rel * mouse_config.sensitivity_x,
+					  y_rel * mouse_config.sensitivity_y,
+					  (x_abs - mouse_video.clip_x) / (mouse_video.res_x - 1) * mouse_config.sensitivity_x,
+					  (y_abs - mouse_video.clip_y) / (mouse_video.res_y - 1) * mouse_config.sensitivity_y,
+					  is_captured);
+
+	MouseSER_NotifyMoved(x_rel, y_rel);
+}
+
+void Mouse_NotifyMovedVMW() {
+
+	AddEvent(DOS_EV::MOUSE_HAS_MOVED);
+}
+
+void Mouse_EventPressed(uint8_t idx) {
+    uint8_t buttons_12S_old = buttons_12 + (buttons_345 ? 4 : 0);
+
+    if (idx < 2) {
+        // left/right button
+        if (buttons_12 & KEY_MASKS[idx]) return;
+        buttons_12 |= KEY_MASKS[idx];
+    } else if (idx < 5) {
+        // middle/extra button
+        if (buttons_345 & KEY_MASKS[idx]) return;
+        buttons_345 |= KEY_MASKS[idx];
+    } else
+        return; // button not supported
+
+    mouse.buttons    = buttons_12 + (buttons_345 ? 4 : 0);
+    bool changed_12S = (buttons_12S_old != mouse.buttons);
+    
+    if (changed_12S) {
+    	uint8_t idx_12S = idx < 2 ? idx : 2;
+    	mouse.buttons   = buttons_12 + (buttons_345 ? 4 : 0);
+
+    	MouseVMW_NotifyPressedReleased(mouse.buttons);
+        MouseSER_NotifyPressed(mouse.buttons, idx_12S);
+
+	 	mouse.times_pressed[idx_12S]++;
+		mouse.last_pressed_x[idx_12S] = POS_X;
+		mouse.last_pressed_y[idx_12S] = POS_Y;
+        AddEvent(SelectEventPressed(idx, changed_12S));
+    }
+}
+
+void Mouse_EventReleased(uint8_t idx) {
+    uint8_t buttons_12S_old = buttons_12 + (buttons_345 ? 4 : 0);
+
+    if (idx < 2) {
+        // left/right button
+        if (!(buttons_12 & KEY_MASKS[idx])) return;
+        buttons_12 &= ~KEY_MASKS[idx];
+    } else if (idx < 5) {
+        // middle/extra button
+        if (!(buttons_345 & KEY_MASKS[idx])) return;
+        buttons_345 &= ~KEY_MASKS[idx];
+    } else
+        return; // button not supported
+
+    mouse.buttons    = buttons_12 + (buttons_345 ? 4 : 0);
+    bool changed_12S = (buttons_12S_old != mouse.buttons);
+
+    if (changed_12S) {
+    	uint8_t idx_12S = idx < 2 ? idx : 2;
+    	
+    	MouseVMW_NotifyPressedReleased(mouse.buttons);
+        MouseSER_NotifyReleased(mouse.buttons, idx_12S);
+
+		mouse.times_released[idx_12S]++;	
+		mouse.last_released_x[idx_12S] = POS_X;
+		mouse.last_released_y[idx_12S] = POS_Y;
+        AddEvent(SelectEventReleased(idx, changed_12S));
+    }
+}
+
+void Mouse_EventWheel(int32_t w_rel) {
+    MouseSER_NotifyWheel(w_rel);
+
+    if (mouse.cute_mouse) {
+	    mouse.wheel = std::clamp(w_rel + mouse.wheel, -0x8000, 0x7fff);
+	    mouse.last_wheel_moved_x = POS_X;
+	    mouse.last_wheel_moved_y = POS_Y;
+
+	    AddEvent(DOS_EV::WHEEL_HAS_MOVED);
+    }
+}
```

Comment:
Wonky identation/formatting throughout the whole file, and like I mentioned, the function prefix should be all caps per convention (`MOUSE_SomeFunction`).

Reply:
Used the script to correct all the formatting. Naming also corrected.